src/App.vue

```diff
@@ -228,12 +230,28 @@ watch(
   }
 )
 
+// === Score Chords functions ===
+function pushSymbolToChord(index: number) {
```

Comment:
Pushing and pulling symbols from `state.scoreChord` is not robust.
To see this press shift and a key on the QWERTY keyboard. This triggers sustain. Edit the entry corresponding to the sustained note. Repress the key to release sustain. `state.scoreChord.delete(item)` now does nothing and the previous symbol is permanently displayed in the widget.

Ideally the chord shown in the widget would be a `computed` of the notes currently being played.